packages/devextreme/js/__internal/utils/version.ts

```diff
@@ -25,6 +29,45 @@ export function parseVersion(version: string): Version {
   };
 }
 
+export function reportDependentVersion(dependentName: string, version: string): void {
+  dependentVersions.push({
+    dependentName,
+    version,
+  });
+}
+
+export function getDependentVersions(): DependentVersion[] {
+  return dependentVersions;
+}
+
+export function clearDependentVersions(): void {
+  /// #DEBUG
+  dependentVersions.splice(0);
+  /// #ENDDEBUG
+}
+
+export function stringifyVersionList(dependents: DependentVersion[]): string {
+  return dependents
+    .map((dependent) => `${dependent.dependentName}: ${dependent.version}`)
+    .join('\n');
+}
+
+export function versionsEqual(versionAStr: string, versionBStr: string): boolean {
+  const versionAComponents = versionAStr.split('.').map(Number);
```

Comment:
Use `parseVersion` funcion from the 'version' module

Such things shouldn't be duplicated, if one day we need to change the parsing logic - we'll have to search for all these places and likely miss some.